app/views/podcast_planner/_calendar.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="col" data-controller="calendar">
```

Comment:
i essentially wrap the simple calendar with a controller for this partial.

Reply:
Alternatively ... it might cleanup your controller code to have 1 controller _per day_ rather than 1 global.  Then you wouldn't need to do a bunch of parent/child checking - you could just tie the 1 controller explicitly to the parent and form-field elements.